lms/services/roster.py

```diff
@@ -449,7 +449,7 @@ def fetch_canvas_sections_roster(self, lms_course: LMSCourse) -> None:
 
             # We use a set here to avoid any pontential duplicates in the API response.
             student_ids = {
-                str(student["id"]) for student in api_section.get("students", [])
+                str(student["id"]) for student in api_section.get("students", []) or []
```

Comment:
```suggestion
                str(student["id"]) for student in api_section.get("students") or []
```
Technically would be the same, but feel free to ignore

Reply:
Yeah, this is nicer, thanks!